haptic_control:
  ros__parameters:
    channel: "SimpleChannelUDP"
    ff_device_ip_address: "192.168.100.53"
    ff_device_param_file: "/etc/Haption/Connector/desktop_6D_n65.param"
    local_ip_address: "192.168.100.50"
    max_force: 6.0
    # robot position safety limits
    use_bounding_box: false
    use_limits: false
    min_x:  -100.0
    max_x:  100.0
    min_y:  -100.0
    max_y:  100.0
    min_z:  -100.0
    max_z:  100.0
    # min_x:  0.33
    # max_x:  0.48
    # min_y: -0.03
    # max_y:  0.18
    # min_z:  -0.10
    # max_z:  0.13

    tool_link_name: "probe"
    base_link_name: "base_link"
    force_scale : 0.3
